Movied/Movied/Scenes/Home/HomeViewController.swift

```diff
@@ -1,9 +1,229 @@
 import UIKit
 
 final class HomeViewController: UIViewController {
-
+    @IBOutlet private weak var homeTitle: UILabel!
+    @IBOutlet private weak var movieListCollectionView: UICollectionView!
+    @IBOutlet private weak var sortedByTextField: UITextField!
+    
+    private var allMovies: [Movie] = []
+    private var sortType: SortType = .newestToOldest
+    private var pageNumber = 1
+    
+    private let refreshControl = UIRefreshControl()
+    private let pickerView = UIPickerView()
+    private let dispatchGroup = DispatchGroup()
+    private var refreshFooterView: RefreshFooterView?
+    
     override func viewDidLoad() {
         super.viewDidLoad()
+        configView()
+        fetchFirstPageMovies()
+    }
+    
+    //MARK: - Fetch Data
+    private func fetchFirstPageMovies() {
+        showIndicator(true)
+        dispatchGroup.enter()
```

Comment:
Should only use dispatchGroup if have multi parallel threads, as call 2 3 4 apis at same time. 
For this func, can simply add the self?.showIndicator(false) command at first line of this block code or next line of guard let.